<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,2">
          <item>
           <widget class="QPushButton" name="btn_ipf">
            <property name="text">
             <string>输入文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete">
            <property name="text">
             <string>删除选定文件</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="lw_ipf">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,1">
          <item>
           <widget class="QPushButton" name="btn_rawopf">
            <property name="text">
             <string>原目录输出</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_opf">
            <property name="text">
             <string>自定义目录输出</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clear">
            <property name="text">
             <string>清空列表</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="lw_opf">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_format">
        <property name="text">
         <string>输出格式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbb_format">
        <item>
         <property name="text">
          <string>mp4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>avi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mpg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mkv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gif</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>flv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mov</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_thread">
        <property name="text">
         <string>线程数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_thread">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
